content/swiftui/concepts/uikit/uikit.md

```diff
@@ -0,0 +1,22 @@
+---
+Title: 'UIKit'
+Description: 'UIKit is a foundational framework for building user interfaces in iOS, iPadOS, and tvOS applications.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+**UIKit** is a foundational framework for building user interfaces in iOS, iPadOS, and tvOS applications. It provides a comprehensive set of components and tools that enable developers to create visually appealing and interactive user interfaces. UIKit has been the primary framework for building iOS apps for many years and continues to be widely used.
```

Comment:
Can we include something about the released year (2008) and its relationship to objective-c (swift came out after)

Reply:
@gab394

Reply:
ok I'll do that